.github/actions/setup-nvidia/action.yml

```diff
@@ -42,10 +42,10 @@ runs:
                   sudo yum-config-manager --add-repo "${YUM_REPO_URL}"
                   sudo yum install -y \
                     nvidia-docker2 \
```

Comment:
Maybe a cleanup item to follow this up with.... https://github.com/pytorch/test-infra/pull/6637/files#r2102001293

Reply:
I have https://github.com/pytorch/test-infra/pull/6943 and https://github.com/pytorch/pytorch/pull/158684 for that following up